src/fixate/ui_cmdline/cmd_line.py

```diff
@@ -1,67 +1,61 @@
+import traceback
 import sys
+import time
 import textwrap
-import traceback
 from queue import Empty
-from queue import Queue
 from pubsub import pub
-from pynput import keyboard
-import fixate.config
-from fixate.config import RESOURCES
+from .kbhit import KBHit
+from queue import Queue
 from fixate.core.exceptions import UserInputError
+from fixate.core.common import ExcThread
+from fixate.config import RESOURCES
+import fixate.config
 
 cmd_line_queue = Queue()
 wrapper = textwrap.TextWrapper(width=75)
 wrapper.break_long_words = False
 
 wrapper.drop_whitespace = True
 
```

Comment:
This is all just a revert of the changes made during #1.
Approved because it should be identical functionality

Reply:
Do you want these lines included? they are removed in another change you made

Reply:
Keep them removed. The comment was meant to be from the full revert of the changes that Kieran made in #1.